tests/test_SVI.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 
 
-EXAMPLEDIR = Path().absolute() / "local_test_database"
+EXAMPLEDIR = Path("P:/11207949-dhs-phaseii-floodadapt/Model-builder/Delft-FIAT/local_test_database")
```

Comment:
For now it is okay to point to the P-Drive but we should find a better solution for this soon.